lib/autokey/model/abstract_hotkey.py

```diff
@@ -45,7 +45,7 @@ def set_hotkey(self, modifiers, key):
         modifiers.sort()
         self.modifiers = modifiers
         self.hotKey = key
-        if key is not None:
+        if key is not None and TriggerMode.HOTKEY not in self.modes:
```

Comment:
This change specifically breaks some tests relating to creating phrases with duplicate hotkeys (which is forbidden). Normally creating duplicates raises an error, but with this change it does not. Could you please review this again @sebastiansam55